### PR TITLE
[TS-3922] Add independent WebSocket timeouts

### PR DESCRIPTION
**Pull request by zizhong**
This pull request adds websocket timeouts (see TS-3922).

Edited by bgaff.

### CODE REVIEW

**Comment by bgaff** [comment]
@zizhong we need to add docs too, can you please update the records.config documentation. Thanks.

File: iocore/net/UnixNet.cc [hunk 1]
```diff
@@ -96,8 +97,8 @@ class InactivityCop : public Continuation
           NET_SUM_DYN_STAT(keep_alive_queue_timeout_total_stat, diff);
           NET_INCREMENT_DYN_STAT(keep_alive_queue_timeout_count_stat);
         }
-        Debug("inactivity_cop_verbose", "vc: %p now: %" PRId64 " timeout at: %" PRId64 " timeout in: %" PRId64, vc, now,
-              vc->next_inactivity_timeout_at, vc->inactivity_timeout_in);
+        Debug("inactivity_cop_verbose_2", "vc: %p now: %" PRId64 " timeout at: %" PRId64 " timeout in: %" PRId64, vc,
```
Comment (bgaff) [inline iocore/net/UnixNet.cc]: I just realized that in the patch I pasted in TS-3922 I left this debug tag name changed as inactivity_cop_verbose_2, @zizhong can you please correct that for me.

Comment (zizhong) [inline iocore/net/UnixNet.cc]: Docs were updated. Tag name was fixed.

**Comment by bgaff** [comment]
Merged.

**Pull request closed without merging**